plugins/main/public/styles/common.scss

```diff
@@ -1818,6 +1818,12 @@ iframe.width-changed {
   z-index: 9999;
 }
 
+.euiFlyout {
+  .euiPage {
+    background-color: transparent;
+  }
+}
+
```

Comment:
**nitpick**: this could interact with the rest of platform and this could be undesired. In this case, it should ensure the definition affects to the specific view.

Reply:
I have narrowed the scope so that the style only applies to the inventory tab. [cb0ff09](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6868/commits/cb0ff09f42c35743346bb5efcdd52b8428b2eb8a)

Reply:
Fixed in commit [edb1225](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6868/commits/edb12255cd89ad4fe39b87f4e2adf14426ad2957)